### PR TITLE
Refactor code and new features added

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
Refactor code and new features added

### CODE REVIEW

File: src/my-web/useEffect/useEffectCart.jsx [hunk 1]
```diff
@@ -142,18 +143,24 @@ export async function handleRegistration(data){
...
-        }).then((res)=>
-             console.log(res)
-        );
-      } catch (error) {
-          console.log(error)
-      }
+    const navigator = useNavigate();
+    
+    // if(email==="adarshbalika@gmail.com" || password==="adarshbalika"){
+    //     return true
```
Comment (anshtripathi01) [inline src/my-web/useEffect/useEffectCart.jsx:151]: Please remove this comment if it's not required

**Review by anshtripathi01 — APPROVED** [review]
Make above changes otherwise codes and format looks good.

File: src/my-web/useEffect/useEffectCart.jsx (hunk 1 above)
Comment (chanchal-panpaliya) [reply src/my-web/useEffect/useEffectCart.jsx]: done

**Review by way2nizam — APPROVED** [review]
Try to create small pull request, 42 files is a large number to review.

**Pull request merged**